practices/java-web/HelloWorldFilterWebApplication/src/main/java/com/dowa/java/web/filters/ParameterFilter.java

```diff
@@ -0,0 +1,31 @@
+package com.dowa.java.web.filters;
+import javax.servlet.*;
+import javax.servlet.http.HttpServletRequest;
+import java.io.IOException;
+/**
+ * Created by nexuz on 31/05/15.
+ */
+public class ParameterFilter implements Filter {
+    @Override
+    public void init(FilterConfig filterConfig) throws ServletException {
+        System.out.println("ParameterFilter init");
+    }
+
+    @Override
+    public void doFilter(ServletRequest servletRequest, ServletResponse servletResponse, FilterChain filterChain)
+            throws IOException, ServletException{
+        System.out.println("ParameterFilter do filter");
+
+        if (servletRequest instanceof HttpServletRequest) {
+            //StringBuffer requestURL = ((HttpServletRequest) servletRequest).getRequestURL();
+            String name = servletRequest.getParameter("name");
```

Comment:
this is too specific since I need to know the name of the parameter to log it, the filter is supposed to log all parameters not just one even new ones servletRequest has a method that allows you to query all the parameter names, then just iterate through them